website/packages/docs/src/pages/how-it-works.mdx

```diff
@@ -0,0 +1,163 @@
+---
+order: 3
+section: 1-Getting started
+---
+
+# How it Works
+
+Once you've [installed and configured](/installation) everything, you can start using it,
+but it's best to explain what happens under the hood.
+
+Compiled uses Babel (with pre-configuration options via Webpack and Parcel) to transform styles
+into atomic CSS determined at build time, resulting in resulting in **very performant components**.
+
+Let's take a typical component you might write:
+
+```jsx
+/** @jsxImportSource @compiled/react */
+import { css } from '@compiled/react';
+import type { ReactNode } from 'react';
+
+type Props = { inverted?: boolean, children: ReactNode };
+
+const largeTextStyles = css({
+  fontSize: '48px',
+  padding: '8px',
+  background: '#eee',
+});
+const invertedStyles = css({
+  background: '#333',
+  color: '#fff',
+});
+
+export const LargeText = ({ inverted, children }: Props): JSX.Element => {
+  return <span css={[largeTextStyles, inverted && invertedStyles]}>{children}</span>;
+};
+```
+
+## Runtime styles (unperformant)
+
+With the most basic Babel setup, Compiled will be transformed to **runtime styles**.
+This is typically used in a development or testing environments.
+
+It is **not recommended in production environments** as it will be less performant, but also mixing
+runtime and extraction can result in breaking visual changes from atomic specificity conflicts.
+Refer to [extracted styles](#extracted-styles) below for the recommendation.
+
+```bash
+$ babel src
+Successfully compiled 1 file with Babel (9ms)
+```
+
+You'll see a few things happening in the code below:
+
+1. All styles are converted into [Atomic CSS](/atomic-css) for reuse.
+2. Some styles, such as `padding:8px` will be expanded out to `padding-left`, `padding-right`, etc,
```

Comment:
IIRC there may be some additional optimitation and certain units could be converted to a more performant unit. Eg px to pc?

Not sure if we should mention this as well

Reply:
Yeah, maybe worth adding just generally:
> 5. Postcss, autoprefixer, and some other optimizations will run over your code to make it more performant and safer for usage.